From: Randy Dunlap <rdunlap@infradead.org>
Subject: mm/migrate: add missing kernel-doc for @migrate_info
Date: Thu, 16 Jul 2026 19:21:46 -0700

Add a kernel-doc comment for @migrate_info so that kernel-doc won't
produce a warning about it:

WARNING: include/linux/mm_types.h:507 struct member 'migrate_info' not described in 'folio'

Link: https://lore.kernel.org/20260717022146.1290242-1-rdunlap@infradead.org
Fixes: f3768940afdc ("mm/migrate: use migrate_info field instead of private")
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: SJ Park <sj@kernel.org>
Reviewed-by: Shivank Garg <shivankg@amd.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/mm_types.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/mm_types.h~mm-migrate-use-migrate_info-field-instead-of-private-fix
+++ a/include/linux/mm_types.h
@@ -368,6 +368,8 @@ typedef unsigned short mm_id_t;
  *    dax_associate_entry.
  * @private: Filesystem per-folio data (see folio_attach_private()).
  * @swap: Used for swp_entry_t if folio_test_swapcache().
+ * @migrate_info: Stores migration state (anon_vma pointer and
+ *    FOLIO_WAS_* markers).
  * @_mapcount: Do not access this member directly.  Use folio_mapcount() to
  *    find out how many times this folio is mapped by userspace.
  * @_refcount: Do not access this member directly.  Use folio_ref_count()
_
